package.json

```diff
@@ -15,6 +15,7 @@
     "minecraft-data": "^3.15.0",
     "mineflayer-pathfinder": "^2.3.3",
     "prismarine-viewer": "^1.23.0",
+    "rg-match-info": "../RegressionGames/GameMachine/rg-match-info",
```

Comment:
We can add this package to NPM. I think we may have to pay for the hosting, and they do have the option for private repos, which may be good in the meantime.